tools/workspace/python/repository.bzl

```diff
@@ -49,6 +49,7 @@ _VERSION_SUPPORT_MATRIX = {
     "ubuntu:18.04": ["3.6"],
     "ubuntu:20.04": ["3.8"],
     "macos": ["3.9"],
+    "manylinux": ["3.6"],
```

Comment:
BTW, okay for now, but eventually we'll want to make this `["3.6", "3.7", "3.8"]`. (As discussed on Slack, there are other changes that are going to be needed, so I think we should leave this as-is for now. This is just a "heads up".)